docs/source/tutorials/token_classification/tutorial-p2.ipynb

```diff
@@ -0,0 +1,657 @@
+    "show_frequent_issues(frequency, entities, verbose=True) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d6d03fa5",
+   "metadata": {},
+   "source": [
+    "As shown above, many mislabels are between `O` and `MISC`, which are inherently hard to differentiate. Therefore, you can add label/prediction pairs to `exclude`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "fda2d30f",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "'Czech' is mislabeled 13 times\n",
+      "-----------------------------\n",
+      "labeled as LOC, but predicted as MISC 12 times\n",
+      "labeled as ORG, but predicted as LOC 1 times\n",
+      "\n",
+      "'National' is mislabeled 8 times\n",
+      "-----------------------------\n",
+      "labeled as ORG, but predicted as MISC 8 times\n",
+      "\n",
+      "'I' is mislabeled 6 times\n",
+      "-----------------------------\n",
+      "labeled as ORG, but predicted as O 5 times\n",
+      "labeled as MISC, but predicted as ORG 1 times\n",
+      "\n",
+      "'Union' is mislabeled 6 times\n",
+      "-----------------------------\n",
+      "labeled as ORG, but predicted as MISC 6 times\n",
+      "\n",
+      "'Hockey' is mislabeled 5 times\n",
+      "-----------------------------\n",
+      "labeled as ORG, but predicted as MISC 5 times\n",
+      "\n",
+      "'United' is mislabeled 5 times\n",
+      "-----------------------------\n",
+      "labeled as LOC, but predicted as ORG 3 times\n",
+      "labeled as ORG, but predicted as LOC 2 times\n",
+      "\n",
+      "'Fe' is mislabeled 4 times\n",
+      "-----------------------------\n",
+      "labeled as ORG, but predicted as LOC 3 times\n",
+      "labeled as LOC, but predicted as ORG 1 times\n",
+      "\n",
+      "'Wto' is mislabeled 4 times\n",
+      "-----------------------------\n",
+      "labeled as ORG, but predicted as MISC 4 times\n",
+      "\n",
+      "'League' is mislabeled 4 times\n",
+      "-----------------------------\n",
+      "labeled as ORG, but predicted as MISC 2 times\n",
+      "labeled as LOC, but predicted as O 1 times\n",
+      "labeled as LOC, but predicted as ORG 1 times\n",
+      "\n",
+      "'At' is mislabeled 4 times\n",
+      "-----------------------------\n",
+      "labeled as O, but predicted as ORG 4 times\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "frequency = frequent_words(issues, words, labels, pred_probs, exclude=[(0, 1), (1, 0)]) \n",
+    "show_frequent_issues(frequency, entities, verbose=True) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a35ef843",
+   "metadata": {},
+   "source": [
+    "## 5. Find issue sentences with particular word \n",
+    "\n",
+    "If you want to examine a specific token, call `search_token` to return a list of sentence indicies which contain the issue sentence. \n",
```

Comment:
"sentence indicies which contain the issue sentence"
->
" indices corresponding to sentences that contain issues pertaining to a particular token"